nucliadb_models/src/nucliadb_models/labels.py

```diff
@@ -35,6 +35,7 @@
     "m": [],  # origin metadata in the form of (key/value). Max key/value size is 255
     "p": [],  # origin metadata in the form of (key/value). Max key/value size is 255
     "k": [],  # kind of text paragraph to be stored
+    "q": [],  # reserved for internal use: h (hidden)
```

Comment:
**i**nternal and **r**eserved were my first choices. Then I thought that there aren't too many one-letter prefix to go around and they are kind of user-visible. So since this one is internal I chose `q` since it's the least common letter in english, and that leaves the more commonly used letters free for more visible usages.